### PR TITLE
More documentation. And fix for dns zone variable for OSP

### PR DESCRIPTION
**Pull request by wkulhanek**
Much more documentation. Final step failed because of unset dns variable.

### CODE REVIEW

File: ansible/configs/ocp4-cluster/destroy_env.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     - name: Remove DNS entry for OpenShift API and ingress
       nsupdate:
         server: "{{ osp_cluster_dns_server }}"
-        zone: "{{ osp_cluster_dns_zone }}"
+        zone: "{{ cluster_dns_zone }}"
```
Comment (fridim) [inline ansible/configs/ocp4-cluster/destroy_env.yml]: what is the point to proxy `osp_cluster_dns_zone` with `cluster_dns_zone` ?

Comment (wkulhanek) [reply ansible/configs/ocp4-cluster/destroy_env.yml]: Continuity?

Comment (tonykay) [reply ansible/configs/ocp4-cluster/destroy_env.yml]: Default it, I like the idea of having non-cloud provider vars where possible.

`
zone: "{{ cluster_dns_zone | default(osp_cluster_dns_zone) }}"
`

File: ansible/configs/ocp4-cluster/post_software.yml [hunk 2]
```diff
@@ -259,7 +259,7 @@
       msg: "{{ item }}"
     loop:
     - "user.info: You can access your bastion via SSH:"
-    - "user.info: ssh {{ student_name }}@bastion.{{ guid }}.{{ cluster_dns_zone }}"
+    - "user.info: ssh {{ student_name }}@bastion.{{ guid }}.{{ ocp4_base_domain }}"
```
[the remark this answers is not in the document]
Comment (wkulhanek) [reply ansible/configs/ocp4-cluster/post_software.yml]: Yeah. I think @vincepower introduced the cluster_dns_zone. We had been using ocp4_base_domain. osp_cluster_dns_zone is OSP centric. We can probably consolidate on cluster_dns_zone. Which may break other configs if we update the secrets.

**Pull request closed without merging**